recipes/junit-xml/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "junit-xml" %}
+{% set version = "1.8" %}
+{% set sha256 = "602f1c480a19d64edb452bf7632f76b5f2cb92c1938c6e071dcda8ff9541dc21 " %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/j/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} setup.py install"
```

Comment:
We try to [use `pip`]( https://conda-forge.org/docs/meta.html#use-pip ) these days when possible.